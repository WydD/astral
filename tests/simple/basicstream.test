<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query t0="auto">
<source attributes="id,T,PHYID,temp" id="S" sourceAttributes="id,T,PHYID,temp" type="Stream"/>
</query>
<sources>
<source attributes="id,T,PHYID,temp" file="tests/simple/basicstream.S" id="S" impl="FileStreamImpl" type="Stream"/>
</sources>
</astral>
