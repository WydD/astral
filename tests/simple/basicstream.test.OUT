id,temp,T
1,30,1000
1,35,1005
1,25,1005
1,29,1005
1,33,1010
1,25,1015
1,25,1015
1,32,1015
1,35,1020
1,27,1020
1,27,1025
1,35,1025
1,27,1035
